README.rst

```diff
@@ -9,7 +9,7 @@ Installation instructions
 
 Create a conda environment for labadmin::
 
-   conda create -n labadmin psycopg2 python=2 click requests
+   conda create -n labadmin psycopg2 python=2 click requests bioconda java-jdk
```

Comment:
no. I don't have the time for this right now. @EmbrietteH needs me to work on the metadata pulldown. My google hit's indicated that the best channel is bioconda. But there might be more out there.
I thought using conda is kind of the lab standard ?!